core/src/main/java/org/moditect/jfrunit/JfrEventsAssert.java

```diff
@@ -29,6 +33,53 @@ public static JfrEventsAssert assertThat(JfrEvents actual) {
         return new JfrEventsAssert(actual);
     }
 
+    public JfrEventsAssert contains(JfrEventType jfrEventType) {
+        AtomicReference<Supplier> errorMessage = new AtomicReference<>(null);
+        AtomicBoolean eventFound = new AtomicBoolean();
+
+        actual.events()
+                .anyMatch(recordedEvent -> {
```

Comment:
I don't think `anyMatch()` should be used with side-effects. Rather assign its result to some boolean and act on that?